src/com/lftechnology/java/training/krishna/employeemanagement/utils/ConstantUtils.java

```diff
@@ -0,0 +1,21 @@
+
+package com.lftechnology.java.training.krishna.employeemanagement.utils;
+/**
+ * Constants.
+ *
+ * @author Krishna Timilsina <krishnatimilsina@lftechnology.com>
+ * @version 1.0
+ */
+public class ConstantUtils {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add a private constructor to hide the implicit public one. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1118)